.travis.yml

```diff
@@ -42,15 +42,14 @@ before_install:
     - export PATH=$PWD:$PATH
     - mkdir deps
     - ln -s /usr/share/perl5/PgCommon.pm deps/
-    - export PATH=/usr/lib/jvm/java-8-oracle/jre/bin/:$PATH
-    - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
     - java -version
-    - ant -version
+    - mvn -version
     - mysql --version
     - sqlite3 --version
     - env
     - cd wrappers/java
-    - ant clean jar
+    - mvn clean
+    - mvn install
```

Comment:
Should be `mvn package`, like in the wrapper

Reply:
Here we need to run tests as well, and in wrapper only create a jar, that's why here is install.

Reply:
No, tests are run in `scripts/dev/travis_run_tests.sh`. This line is to pre-compile the java code outside of the test framework, which makes it easier to debug